examples/cms-contentstack/components/cover-image.js

```diff
@@ -0,0 +1,25 @@
+import cn from 'classnames'
+import Link from 'next/link'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <img
+      src={url}
+      alt={`Cover Image for ${title}`}
+      className={cn('shadow-small', {
+        'hover:shadow-medium transition-shadow duration-200': slug,
+      })}
+    />
+  )
+  return (
+    <div className="sm:mx-0">
+      {slug ? (
+        <Link as={`/posts/${slug}`} href="/posts/[slug]">
```

Comment:
"as" is no longer needed! You can convert as->href and remove what you have from href here.